src/datepicker/datepicker.js

```diff
@@ -472,16 +480,29 @@ angular.module('mgcrea.ngStrap.datepicker',
 					return;
 				}
 
+				function triggerValid() {
```

Comment:
These two helper methods simply check if the callback was actually defined, and if it is, call it